dGame/dComponents/ControllablePhysicsComponent.cpp

```diff
@@ -33,6 +33,10 @@ ControllablePhysicsComponent::ControllablePhysicsComponent(Entity* entity) : Com
 	m_PickupRadius = 0.0f;
 	m_DirtyPickupRadiusScale = true;
 	m_IsTeleporting = false;
+	m_AttachedPath = entity->GetVarAsString(u"attached_path");
+	m_PathWaypoint = entity->GetVarAs<int>(u"attached_path_start");
+	m_PathSpeed = 2.5f;
```

Comment:
The PhysicsComponent has a speed variabel. Check the movement ai component.

Reply:
is that the same one from the movementAI component? I know the controllable physics comp has a speed varaible, but it's not read in here. I plan on reading it in and using it evetually, but didn't want to go down that rabbit hole yet, since most of the have a defined speed on their path waypoint configs